package.json

```diff
@@ -41,9 +41,11 @@
     "request": "^2.60.0"
   },
   "devDependencies": {
+    "babel-eslint": "^4.1.1",
```

Comment:
added eslint for linting

I didn't push my .eslintrc (which configures linting). I can if you all would like that.